modules/landcall.js

```diff
@@ -1090,40 +1125,68 @@ while(empty.length>0){
     let tempRan = Math.floor(Math.random()*empty4.length)-1;
     let temp=empty4.splice(tempRan,1);
 
-    section[5][temp[0][0]][temp[0][1]]=[46,1,[[[],[],"DOWNSTAIRS ENTRANCE",false,[],[]]]];
-    section[8][temp[0][0]][temp[0][1]]=[46,1,[[[],[],"DOWNSTAIRS ENTRANCE",false,[],[]]]];
-    section[6][temp[0][0]][temp[0][1]]=[47,1,[[[],[],"ASCENDING STAIRS",false,[],[]]]];
-    section[9][temp[0][0]][temp[0][1]]=[47,1,[[[],[],"ASCENDING STAIRS",false,[],[]]]];
+    section[5][temp[0][0]][temp[0][1]]=generateBasicTile(46, "DOWNSTAIRS ENTRANCE");
+    section[8][temp[0][0]][temp[0][1]]=generateBasicTile(46, "DOWNSTAIRS ENTRANCE");
+    section[6][temp[0][0]][temp[0][1]]=generateBasicTile(47, "ASCENDING STAIRS");
+    section[9][temp[0][0]][temp[0][1]]=generateBasicTile(47, "ASCENDING STAIRS");
 
     empty5.splice(empty5.findIndex(tile => tile[0] == temp[0][0] && tile[1] == temp[0][1]),1)
   }
+
 //tossing in the sac slabs, stealing the light aspect symbol for now.
   let tempRan = Math.floor(Math.random()*empty4.length)-1;
   let temp=empty4.splice(tempRan,1);
-  section[6][temp[0][0]][temp[0][1]]=[35,1,[[[],[],"SACRIFICIAL SLAB",false,[],[]]]];
-  section[9][temp[0][0]][temp[0][1]]=[35,1,[[[],[],"SACRIFICIAL SLAB",false,[],[]]]];
+  section[6][temp[0][0]][temp[0][1]]=generateBasicTile(35, "SACRIFICIAL SLAB");
+  section[9][temp[0][0]][temp[0][1]]=generateBasicTile(35, "SACRIFICIAL SLAB");
   empty5.splice(empty5.findIndex(tile => tile[0] == temp[0][0] && tile[1] == temp[0][1]),1)
 
   while(empty3.length>0){
     let temp=empty3.splice(Math.floor(Math.random()*empty3.length)-1,1);
-    section[4][temp[0][0]][temp[0][1]]=[15,1,[[[],[],"PRISON CELL",false,[],[]]]];
-    section[7][temp[0][0]][temp[0][1]]=[15,1,[[[],[],"PRISON CELL",false,[],[]]]];
+    section[4][temp[0][0]][temp[0][1]]=generateBasicTile(15, "PRISON CELL");
+    section[7][temp[0][0]][temp[0][1]]=generateBasicTile(15, "PRISON CELL");
   }
   while(empty4.length>0){
     let temp=empty4.splice(Math.floor(Math.random()*empty4.length)-1,1);
-    section[5][temp[0][0]][temp[0][1]]=[15,1,[[[],[],"PRISON CELL",false,[],[]]]];
-    section[8][temp[0][0]][temp[0][1]]=[15,1,[[[],[],"PRISON CELL",false,[],[]]]];
+    section[5][temp[0][0]][temp[0][1]]=generateBasicTile(15, "PRISON CELL");
+    section[8][temp[0][0]][temp[0][1]]=generateBasicTile(15, "PRISON CELL");
   }
   while(empty5.length>0){
     let temp=empty5.splice(Math.floor(Math.random()*empty5.length)-1,1);
-    section[6][temp[0][0]][temp[0][1]]=[15,1,[[[],[],"PRISON CELL",false,[],[]]]];
-    section[9][temp[0][0]][temp[0][1]]=[15,1,[[[],[],"PRISON CELL",false,[],[]]]];
+    section[6][temp[0][0]][temp[0][1]]=generateBasicTile(15, "PRISON CELL");
+    section[9][temp[0][0]][temp[0][1]]=generateBasicTile(15, "PRISON CELL");
   }
 
   return section;
 }
 
 
+function generateEmptyLine(name, tile, length = 11){
+	let retVal = [];
+	for(let i=0; i<length; i++){
+		retVal.push(generateBasicTile(10, name));
```

Comment:
don't you want to use tile instead of 10 here, so you can make different types of basic empty tiles?